distributed/http/scheduler/tests/test_scheduler_http.py

```diff
@@ -245,3 +246,51 @@ async def test_eventstream(c, s, a, b):
     )
     assert "websocket" in str(s.plugins).lower()
     ws_client.close()
+
+
+@gen_cluster(client=True, clean_kwargs={"threads": False})
+async def test_api(c, s, a, b):
+    http_client = AsyncHTTPClient()
+
+    response = await http_client.fetch(
+        "http://localhost:%d/api/v1" % s.http_server.port
+    )
```

Comment:
In the other tests you are using `aiohttp`. Perhaps it would be good to use that here for consistency.